src/Lykke.Service.Balances/Workflow/Handlers/TotalBalanceCommandHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Lykke.Common.Chaos;
+using Lykke.Cqrs;
+using Lykke.Service.Balances.Core.Services;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.Balances.Workflow.Handlers
+{
+    public class TotalBalanceCommandHandler
+    {
+        private readonly IChaosKitty _chaosKitty;
+        private readonly ITotalBalancesService _totalBalancesService;
+
+        public TotalBalanceCommandHandler(IChaosKitty chaosKitty, ITotalBalancesService totalBalancesService)
+        {
+            _chaosKitty = chaosKitty;
+            _totalBalancesService = totalBalancesService;
+        }
+
+        public async Task<CommandHandlingResult> Handle(Commands.UpdateTotalBalanceCommand command)
+        {
+            await _totalBalancesService.ChangeTotalBalanceAsync(command.AssetId, command.BalanceDelta,
```

Comment:
Any events deduplication?

Reply:
I don't see how it can be helpful here. We either incrementing value in Redis, or having a network problems with Redis, and doing a retry later. So no need for it from my point of view.